frontend/src/components/Dashboards/DashboardNavbar.js

```diff
@@ -102,19 +103,39 @@ class MenuAppBar extends React.Component {
     settings: false
   };
 
+  getLeagues() {
+    const token = localStorage.getItem('jwt') || this.props.context.signOut();
+    const endpoint = 'http://localhost:4000/leagues/';
```

Comment:
The endpoint needs to point to the hosted server, rather than the local server.

Reply:
Should work now 👍